app/views/_includes/forms/trainee-id.html

```diff
@@ -5,13 +5,11 @@
     isPageHeading: true
   },
   hint: {
-    text: "Your internal identifier for this trainee or candidate"
+    text: "Assign your own ID for this trainee to help you identify them and search for them in your trainee records. You'll also use it if you get in touch with DfE about this trainee."
```

Comment:
```suggestion
    text: "Assign your own ID for this trainee to help you identify them and search for them in your trainee records. You’ll also use it if you get in touch with DfE about this trainee."
```

I'm not sure the second sentence is quite accurate - it's more that DfE may use it if contacting them. But I suspect they might use the trn if it's available.